server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/impl/TopicDao.java

```diff
@@ -46,6 +46,15 @@
     List<T> findTopicsByAppIdAndType(String appId, TopicTypeDto typeDto);
 
     /**
+     * Find topics by application id and name.
+     *
+     * @param appId the application id
+     * @param topicName the topic name
+     * @return the list of topics
+     */
+    List<T> findTopicsByAppIdAndName(String appId, String topicName);
```

Comment:
Why this method returns list if we expect unique topic for application id and name?